ci/version.sh

```diff
@@ -1,5 +1,5 @@
 if [ -z "$TRAVIS_TAG" ]
 then
     HASH=$(git log --pretty=format:'%h' -n 1)
-    sed -i -E "s/version = \"([^\"]+?)\"$/version = \"\1-$HASH\"/" Cargo.toml
+    sed -E "s/version = \"(.+)\"$/version = \"\1-$HASH\"/" Cargo.toml > Cargo.toml.versioned && mv Cargo.toml.versioned Cargo.toml
```

Comment:
GNU sed and BSD sed works differently, especially the `-i` argument. AFAIK this is the simplest way to replace content inside the file being read, writing stdout directly to it sadly results in an empty file